python/dolfin/mesh.py

```diff
@@ -10,6 +10,7 @@
 # __all__ = ["MeshFunction", "MeshValueCollection"]
 
 _meshfunction_types = {
+    "bool": cpp.mesh.MeshFunctionbool,
```

Comment:
I thought we had remove MeshFunction<bool> deliberately?

Reply:
MeshValueCollection and MeshFunction stayed. But I think specialization for bools shouldnt exist, it is a leftover probably.

Reply:
I can remove the MeshFunctionbool for the python exposure and tests

Reply:
Removed.